src/main/java/be/seeseemelk/mockbukkit/entity/SimpleMobMock.java

```diff
@@ -2,6 +2,7 @@
 
 import be.seeseemelk.mockbukkit.ServerMock;
 import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
```

Comment:
Where is this Import Used. Seams to be unnecessary if it's only imported but not used

Reply:
Good 'ol merge conflicts